lib/pages/creature-page.tsx

```diff
@@ -327,7 +330,10 @@ const AutoSizingSvg = React.forwardRef(
         viewBox={`${x} ${y} ${width} ${height}`}
         ref={ref}
       >
-        {props.children}
+        {bgColor && (
+          <rect x={x} y={y} width={width} height={height} fill={bgColor} />
+        )}
+        <g ref={gRef}>{props.children}</g>
```

Comment:
We need to wrap the actual children in `<g>` and use that as the bounding box to crop the image around--otherwise if we take the new `<rect>` into account it will cause a resizing loop that grows the image infinitely.